### ISSUE TITLE
theme: disable mouse over icon highlighting

### ISSUE THREAD

**Issue by star-buck** [blue-systems/netrunner-next#140]
Netrunner on my laptop has these icons not doing anything when moving the cursor over them, even though the theme is breeze. there were settings in Plasma at some point,  i guess these are just hidden for now but still apparently configurable

also the outlined frame of the main icon area turning blue when in focus, it is not doing that here.

these two should be configured to be OFF.

**Comment by star-buck** [comment]
@davidedmundson adding David, as I recall you helped with making that configuration once in the past

**Comment by notmart** [comment]
To confirm we undertood the issue, here is a screenshot of Dolphin from netrunner 23 live image:

![image](https://github.com/blue-systems/netrunner-next/assets/4526371/2312f643-8b2d-4f57-bb5e-01d199a76984)

so, the icon "public" under the mouse is highlighted , that should not do it, and also the blue rectangle of the icon area border should not appear, correct?

**Comment by notmart** [comment]
@UriHerrera do you know the package versions of frameworks/plasma/applications of the new release in progress?

**Comment by UriHerrera** [comment]
> @UriHerrera do you know the package versions of frameworks/plasma/applications of the new release in progress?

Plasma 5.27.5
KDE Apps 22.12.3
Frameworks 5.103.0
Qt 5.15.8

**Comment by davidedmundson** [comment]
To disable highlight effect on icons add the following to kdeglobals

```
[DesktopIcons]
ActiveEffect=none

[PanelIcons]
ActiveEffect=none
```

It will stop the icon changing brightness, the text label still shows a mouse over colour.  Will wait for confirmation of Marco's comment above then look into that if there's work to be done.

**Comment by davidedmundson** [comment]
In the past, we had a fork of Oxygen called "Carbon" which was how we removed the other hover effects on the frames. 
If we're still using Carbon, that will need an update.

**Comment by notmart** [comment]
> > @UriHerrera do you know the package versions of frameworks/plasma/applications of the new release in progress?
> 
> Plasma 5.27.5 KDE Apps 22.12.3 Frameworks 5.103.0 Qt 5.15.8

ok, thanks.

so, if carbon is still used that should be set as default as it has some of the wanted changes, otherwise we might need to package a fork of breeze that we can patch with the desired things (btw the 6.0 version of breeze doesn't have that blue focus rectangle anymore)

there might be some other minor patches in frameworks needed to be done if we wish to thoroughly remove all the  hover effects all around

**Comment by star-buck** [comment]
yes, but here it is not carbon, but breeze.... not sure how James did it he is no longer active here.

Now that I have unfortunately "reset" the global theme, that frame is also turning blue again (not sure why it wasnt before), which is what should not happen, but the icons on hover are not changing, which is fine.

See these two pictures:
![Screenshot_20240603_113041](https://github.com/blue-systems/netrunner-next/assets/840070/189e554c-6c05-4836-af98-e05d18d6b8b7)
in this the focus and mouse hover is above the main dolphin area where icons are shown, so the icon is not highlighted

when moving the cursor on the icon, the icon does still not highlight:
![Screenshot_20240603_113212](https://github.com/blue-systems/netrunner-next/assets/840070/33c1ae66-2c34-49d3-9aee-423c95155ad0)

Everything else is fine, so this is not about the frame around the folder name.

**Comment by star-buck** [comment]
this is the version installed here:
![Screenshot_20240603_114337](https://github.com/blue-systems/netrunner-next/assets/840070/bc7e55d2-7ca2-4b98-8a86-4097b3f48671)

**Comment by star-buck** [comment]
there are also 3 colors, non highlighted/non-active, highlighted and active:
non-highlighted:
![Screenshot_20240603_114913](https://github.com/blue-systems/netrunner-next/assets/840070/17690579-4943-49c6-b307-9696dd0f8d22)
highlighted:
![Screenshot_20240603_114951](https://github.com/blue-systems/netrunner-next/assets/840070/c2805d18-b5a5-4ef9-881c-c4db60a0916f)
active:
![Screenshot_20240603_115555](https://github.com/blue-systems/netrunner-next/assets/840070/6e8458c0-2138-4fe3-8b76-d153d3d64f63)

best would be if all 3 colors would be the same grey than the non-highlighted-nonactive

**Comment by star-buck** [comment]
Okay, I found out that for the frame line, James likely used a color scheme:
Uising "Krita Dark" for example has only the highlight color, but active and non-active seem the same.
Whats further nice is that the "extra line" directly below a toolbar seems gone too, aka "color matched".
Compare Breeze Dark
![Screenshot_20240603_120419](https://github.com/blue-systems/netrunner-next/assets/840070/3bf179d3-d253-425e-b26c-6addf2c7b464)

vs Krita Dark:
![Screenshot_20240603_120342](https://github.com/blue-systems/netrunner-next/assets/840070/72edfebf-281f-4cb1-870c-171a6c9ad934)

**Comment by star-buck** [comment]
So above Krita Dark is already much more like it should be for Netrunner. We just need to find out what colors to manipulate in "Breeze Dark" to make it less obstrusive, then use that as "Netrunner Dark" color scheme.

As for the icon not highlighting, I am sure it was an accessible setting for a long time which I can no longer find though via Settings GUI. It must be still configurable or otherwise the icons would still highlight when cursor hovers over them.
